src/storage-operators/src/persist_source.rs

```diff
@@ -104,9 +104,30 @@ pub fn persist_source<G>(
 where
     G: Scope<Timestamp = mz_repr::Timestamp>,
 {
+    // WIP is this actually equivalent to None?
+    let max_inflight_bytes = max_inflight_bytes.unwrap_or(usize::MAX);
```

Comment:
in storage we just dont render the `backpressure` operator if its `None`, but `MAX` here will be very similar

Reply:
yeah, was just being lazy :D

Reply:
Done!